backend/app/Http/Requests/UserUpdateInfoRequest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UserUpdateInfoRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
```

Comment:
do checks here,

1 if user doing the request is admin
2 if user to update is same logged in user

if user is admin, no need to check if same logged in user

Reply:
Added logic to match above